### ISSUE TITLE
option named with prefix --no: "no" get stripped away from the parsed options

### ISSUE THREAD

**Issue by dloreto** [tj/commander.js#1806]
i'm creating this options: `.option('--no-cache', 'Do not use cache when building the image', false)`

When launching my command with `--no-cache` flag, the resulting parsed options are: `{ cache: false }`

the "no" prefix as been stripped away.

![image](https://user-images.githubusercontent.com/29567012/192416890-fc334640-9a51-4e5b-ba4a-b5a8c6e39316.png)

This is the command I'm running: `node index.js build-api --no-cache`

**Comment by shadowspawn** [comment]
Yes, that is a feature not a bug. The option name strips the "no" and the option value is true by default. The option value is false when `--no-cache` is specified.

Have a looking the README at the section covering negatable boolean: https://github.com/tj/commander.js#other-option-types-negatable-boolean-and-booleanvalue

**Comment by shadowspawn** [comment]
An answer was provided, and no further activity in a month. Closing this as resolved.

Feel free to open a new issue if it comes up again, with new information and renewed interest.
